src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -111,6 +108,8 @@ public void postConstruct() {
   public void receiveStartJobCommand(JobCommand jobCommand, Acknowledgment acknowledgment) {
     log.info("Received {}.", jobCommand);
 
+    //This thread wrapper is used not to exceed kafka.consumer.properties.max.poll.interval.ms for long-running jobs
+    new Thread(() -> {
```

Comment:
Could you clarify why the whole content of this method is wrapped in a new thread and not just exportJobManagerSync.launchJob(jobLaunchRequest)?

Reply:
You're right, it'd be better to wrap only job launching